platform/viewer/src/connectedComponents/ViewerRetrieveStudyData.js

```diff
@@ -83,18 +85,56 @@ class ViewerRetrieveStudyData extends Component {
     });
   }
 
+  _handleSeriesLoadResult(error, studyMetadata, series) {
+    if (this.abortSeriesLoad) return;
```

Comment:
Are we aborting here because the studyMetadataManager may be dead? If not, why are we?

Reply:
no, because the view might have been unmounted....